packages/trpc/server/routers/viewer/teams/_router.tsx

```diff
@@ -147,7 +147,7 @@ export const viewerTeamsRouter = router({
 
   inviteMember: authedProcedure.input(ZInviteMemberInputSchema).mutation(async ({ ctx, input }) => {
     if (!UNSTABLE_HANDLER_CACHE.inviteMember) {
-      UNSTABLE_HANDLER_CACHE.inviteMember = await import("./inviteMember.handler").then(
+      UNSTABLE_HANDLER_CACHE.inviteMember = await import("./inviteMember/inviteMember.handler").then(
```

Comment:
Moved into a folder so i can colocate tests + utils without creating them in the root team folder